packages/features/ee/organizations/pages/settings/members.tsx

```diff
@@ -0,0 +1,175 @@
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+import MemberInvitationModal from "@calcom/features/ee/teams/components/MemberInvitationModal";
+import MemberListItem from "@calcom/features/ee/teams/components/MemberListItem";
+import TeamInviteList from "@calcom/features/ee/teams/components/TeamInviteList";
+import { getLayout } from "@calcom/features/settings/layouts/SettingsLayout";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { MembershipRole } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc/react";
+import type { RouterOutputs } from "@calcom/trpc/react";
+import { Button, Meta, TextField, showToast } from "@calcom/ui";
+import { Plus } from "@calcom/ui/components/icon";
+
+type Team = RouterOutputs["viewer"]["teams"]["get"];
+
+interface MembersListProps {
+  team: Team | undefined;
+}
+
+const checkIfExist = (comp: string, query: string) =>
+  comp.toLowerCase().replace(/\s+/g, "").includes(query.toLowerCase().replace(/\s+/g, ""));
+
+function MembersList(props: MembersListProps) {
+  const { team } = props;
+  const { t } = useLocale();
+  const [query, setQuery] = useState<string>("");
+
+  const members = team?.members;
+  const membersList = members
+    ? members && query === ""
+      ? members
+      : members.filter((member) => {
+          const email = member.email ? checkIfExist(member.email, query) : false;
+          const username = member.username ? checkIfExist(member.username, query) : false;
+          const name = member.name ? checkIfExist(member.name, query) : false;
+
+          return email || username || name;
+        })
+    : undefined;
+  return (
+    <div className="flex flex-col gap-y-3">
+      <TextField
+        type="search"
+        autoComplete="false"
+        onChange={(e) => setQuery(e.target.value)}
+        value={query}
+        placeholder={`${t("search")}...`}
+      />
+      {membersList?.length && team ? (
+        <ul className="divide-subtle border-subtle divide-y rounded-md border ">
+          {membersList.map((member) => {
+            return <MemberListItem key={member.id} team={team} member={member} />;
+          })}
+        </ul>
+      ) : null}
+    </div>
+  );
+}
+
+const MembersView = () => {
+  const { t, i18n } = useLocale();
+  const router = useRouter();
+  const session = useSession();
+  const utils = trpc.useContext();
+  const showDialog = router.query.inviteModal === "true";
+  const [showMemberInvitationModal, setShowMemberInvitationModal] = useState(showDialog);
+  const { data: team, isLoading } = trpc.viewer.organizations.listMembers.useQuery(undefined, {
+    onError: () => {
+      router.push("/settings");
+    },
+  });
```

Comment:
On delete/edit  in MemberListItem we need to invalidate this item so we dont have to manually refresh to see changes